Build break

With latest Sway and wlroots sources:
```
ninja: Entering directory `build'
[466/712] Compiling C object sway/sway.p/input_text_input.c.o
FAILED: sway/sway.p/input_text_input.c.o
cc -Isway/sway.p -Isway -I../sway -Iinclude -I../include -Isubprojects/wlroots/include -I../subprojects/wlroots/include -Iprotocols -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/json-c -I/usr/include/libevdev-1.0 -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -O0 -g -DWLR_USE_UNSTABLE -D_POSIX_C_SOURCE=200809L -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -Wno-format-zero-length -Wundef -Wvla '-DSYSCONFDIR="//usr/local/etc"' '-DSWAY_VERSION="1.11-rc2-3fbff5b4 (" __DATE__ ", branch '"'"'master'"'"')"' -fmacro-prefix-map=../= -pthread -MD -MQ sway/sway.p/input_text_input.c.o -MF sway/sway.p/input_text_input.c.o.d -o sway/sway.p/input_text_input.c.o -c ../sway/input/text_input.c
../sway/input/text_input.c: In function ‘sway_input_method_relay_init’:
../sway/input/text_input.c:635:50: error: ‘struct <anonymous>’ has no member named ‘text_input’; did you mean ‘new_text_input’?
  635 |         wl_signal_add(&server.text_input->events.text_input,
      |                                                  ^~~~~~~~~~
      |                                                  new_text_input
[483/712] Compiling C object sway/sway.p/commands_bind.c.o
ninja: build stopped: subcommand failed.
```

--- a/sway/input/text_input.c
+++ b/sway/input/text_input.c
@@ -632,7 +632,7 @@ void sway_input_method_relay_init(struct sway_seat *seat,
 	wl_list_init(&relay->input_popups);
 
 	relay->text_input_new.notify = relay_handle_text_input;
-	wl_signal_add(&server.text_input->events.text_input,
+	wl_signal_add(&server.text_input->events.new_text_input,
 		&relay->text_input_new);
 	relay->text_input_manager_destroy.notify = relay_handle_text_input_manager_destroy;
 	wl_signal_add(&server.text_input->events.destroy,

